src/components/core/seo.js

```diff
@@ -62,13 +62,9 @@ function SEO({ desc, lang, meta, keywords, title }) {
           >
             <script type="text/javascript">
               {`
```

Comment:
is {` required?

Reply:
yes, sir, I believe it is, `react-helmet` requires that to be wrapped